client/starwhale/__init__.py

```diff
@@ -10,6 +10,7 @@
     Image,
     Video,
     Binary,
+    JsonDict,
```

Comment:
Is `JsonDict` a common type for users or is it an internal type for starwhale developers?

The current annotations can support users defining a python dict type,  so the `JsonDict` is transparent for the users.

Reply:
Yes, the `JsonDict` is transparent for the users currentlly . The exportation is removed in api module.